docs: improve CRD reference intro paragraphs (#3320)
keptn/lifecycle-toolkit: docs/docs/reference/crd-reference/analysis.md

# Analysis

> An `Analysis` resource is an instance of an

Reviewer: Maybe before this a sentence  on the line of "An Analysis  is a snapshot of your current application status, based on your defined SLIs  it can show if your SLO are satisfied... "

Author: done

[AnalysisDefinition](analysisdefinition.md) resource

> which defined specific data like

Reviewer 2: ```suggestion
which defines specific data like
```

> which defined specific data like

Reviewer: ```suggestion
which defines specific data like
```

# Analysis

An `Analysis` is a snapshot of your current application status.
Based on your defined SLIs, it can validate that your SLOs are satisfied,

> based on the data coming from your defined set of `KeptnMetricsProvider` resources.

Reviewer: ```suggestion
using the data coming from your defined set of `KeptnMetricsProvider` resources.
```
